atdll/atdll.cpp

```diff
@@ -847,6 +955,9 @@ AT_RegisterRawInputDevicesHook(
             debugf("Registering global hotkeys with hWnd=%p", hWnd);
             RegisterHotKey(hWnd, HOTKEY_ENABLE_ID, HOTKEY_ENABLE_MOD, HOTKEY_ENABLE_VK);
             RegisterHotKey(hWnd, HOTKEY_CALIBRATION_ID, HOTKEY_CALIBRATION_MOD, HOTKEY_CALIBRATION_VK);
+            RegisterHotKey(hWnd, HOTKEY_LOAD_ID, HOTKEY_LOAD_MOD, HOTKEY_LOAD_VK);
```

Comment:
Any reason why you explicitly load the config instead of doing it automatically? I think it would be better if we just always try to load the config at startup and do nothing if the file doesn't exist.

Reply:
Good idea, but I'd like to still give the option to reload the config without restarting the game, just in case they accidentally hit the calibration shortcut.